voyager-transitions/src/commonMain/kotlin/cafe/adriel/voyager/transitions/ScreenTransition.kt

```diff
@@ -55,6 +56,26 @@ public fun ScreenTransition(
     )
 }
 
+@Composable
+public fun ScreenTransition(
+    navigator: Navigator,
+    screenTransition: ScreenTransition,
```

Comment:
practice has shown that it is convenient to get a default transition as `ScreenTransition` argument here